lambda/shared/src/main/scala/feral/lambda/IOLambda.scala

```diff
@@ -20,12 +20,18 @@ package lambda
 import cats.effect.IO
 import io.circe.Decoder
 import io.circe.Encoder
+import cats.effect.kernel.Resource
 
 abstract class IOLambda[Event, Result](
     implicit private[lambda] val decoder: Decoder[Event],
     private[lambda] val encoder: Encoder[Result]
 ) extends IOLambdaPlatform[Event, Result]
     with IOSetup {
 
-  def apply(event: Event, context: Context, setup: Setup): IO[Option[Result]]
+  final type Setup = Lambda[IO, Event, Result]
+
+  final override protected def setup: Resource[IO, Setup] = run
+
+  def run: Resource[IO, Lambda[IO, Event, Result]]
```

Comment:
In the spirit of bikeshedding, I like `install` for this better than `run` or `handle`. I think the latter two imply this would run for every invocation of the Lambda, but if I understand the lifecycle correctly, it will really only run during initialization, and subsequent requests will basically go straight to the `Lambda[IO, Event, Result]`.

Reply:
👍 that was exactly my thinking as well, if we think of it as a verb/action ("install the lambda"). If we think of it as a thing/noun then it is the "handler".